Utilities/Docker/Dockerfile

```diff
@@ -20,13 +20,14 @@ ENV LC_ALL en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US.UTF-8
 
-# SwiftPM dependencies
+# SwiftPM dependancies
```

Comment:
It is said that [dependancy](https://en.wiktionary.org/wiki/dependancy) is an Archaic form of [dependency](https://en.wiktionary.org/wiki/dependency#English)
Is there a reason to change this comment?

Reply:
yea that is a typo :)

Reply:
Seems like the newest commits still contains the typo.